src/pages/index.js

```diff
@@ -133,3 +250,21 @@ profileEditButton.addEventListener("click", () => {
 addNewCardButton.addEventListener("click", () => {
   newCardPopup.open();
 });
+
+// edit avatar pop up
+editAvatarButton.addEventListener("click", () => {
+  const currentAvatar = document.querySelector(".profile__image").src;
+  document.querySelector("#edit-avatar-link-input").value = currentAvatar;
+  editAvatarPopup.open();
+});
+
+editAvatarButtonPencil.addEventListener("click", () => {
+  const currentAvatar = document.querySelector(".profile__image").src;
+  document.querySelector("#edit-avatar-link-input").value = currentAvatar;
+  editAvatarPopup.open();
+});
+
+editAvatarForm.addEventListener("submit", (e) => {
+  const formValues = editAvatarPopup._getInputValues();
+  handleAvatarEditSubmit(e, formValues);
```

Comment:
Please, delete it

The avatar form submission should be handled by `PopupWithForm`